Dockerfile-production

```diff
@@ -47,7 +47,7 @@ RUN sudo apt-get install -yq nodejs \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install gem sass for grunt-contrib-sass
+# Install gem sass
 RUN gem install sass
```

Comment:
I am on Win10 with no Sass/Ruby installed and I just did a simple scss file for core and gulp compiled it to css and i see the style change.